content/kubernetes-platform-services/dashboard/cluster-portal.md

```diff
@@ -2,3 +2,144 @@
 title: "Cluster Portal"
 description: ""
 ---
+
+[Cluster portal](https://github.com/cloudposse/charts/tree/master/incubator/portal) allow to access
+internal dashboards after authorization with third party [oAuth](https://en.wikipedia.org/wiki/OAuth) service.
+
+Cluster portal use [oauth2proxy](https://github.com/bitly/oauth2_proxy) as component
+responsible for authentification.
+
+# Dependencies
+
+* [External DNS]({{< relref "kubernetes-backing-services/external-dns/external-dns.md" >}})
+* [Kube Lego]({{< relref "kubernetes-backing-services/tls-management/kube-lego-lets-encrypt.md" >}})
+
+# Install
+
+Starting installation the `portal` you need to define `hostname` used to access the portal.
```

Comment:
To install the `portal`, you need to define `hostname`, which is a DNS name used to access the portal